src/Kiota.Builder/CodeDOM/CodeEnum.cs

```diff
@@ -12,6 +12,13 @@ public bool Flags
     {
         get; set;
     }
+
+    public string Style
```

Comment:
I agree. I had thought about this as well and figured it would be a problem for https://github.com/microsoft/kiota/issues/3238 as we aren't passing the info to the serializers yet. But it better to just do it now.